components/Molecule/navigation-component.html

```diff
@@ -0,0 +1,72 @@
+<div class="component"  title="Navigation Component" version="1">
+
+    <style>
+.navigation-components{
+    margin: 100px 100px;
+}
+
+#main-nav li{
+  text-align: center;
+}
+aside{
+    display: none;
+    background-color: whitesmoke;
+    float: right;
```

Comment:
We are using grid so you can get red of float. Also, it will be display grid later after we add this molecule to the page layout, so you don't need display: none. There is no background color in the original task #87, do you just like to have one?